streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -81,4 +92,44 @@ public void testReprocessingFromDateTimeAfterResetWithoutIntermediateUserTopic()
     public void testReprocessingByDurationAfterResetWithoutIntermediateUserTopic() throws Exception {
         super.testReprocessingByDurationAfterResetWithoutIntermediateUserTopic();
     }
+
+    @Test
+    public void testInputTopicAbsentWithoutIntermediateUserTopic() throws Exception {
```

Comment:
maybe a nit: I understand what you are doing here, but IMHO there should be two separate tests.

Reply:
This test just needs a rename: it calls `shouldNotAllowToResetWhileStreamsIsRunning()` and should have the same name. There are two separate test below: one for invalid input topic and one for invalid intermediate topic. (or did you mean something else, @bbejeck )

Reply:
@mjsax  @bbejeck  I renamed the method. Apologize for my poor method naming skills.

Reply:
No need to apologize. :)

Reply:
@ssanthalingam thanks for the fix, no apologies necessary!